fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulktools/Main.java

```diff
@@ -97,7 +97,7 @@
      * Parse the command line arguments
      *   --cosapikey:       the IBM COS API key or S3 access key.
      *   --cossrvinstid:    the IBM COS service instance id or S3 secret key.
-     *   --cosendpinturl:   the IBM COS or S3 End point URL.
+     *   --cosEndpointUrl:   the IBM COS or S3 End point URL.
      *   --coslocation:     the IBM COS or S3 location.
      *   --cosbucketname:   the IBM COS or S3 bucket name to import from.
      *   --coscredentialibm: if use IBM credential(Y/N), default(Y).
```

Comment:
the mix of all-lower and camelCase looks odd to me

Reply:
I can't speak to it

Reply:
fixed